src/BrefServiceProvider.php

```diff
@@ -71,6 +75,10 @@ public function boot()
                 __DIR__ . '/../stubs/runtime.php' => base_path('php/runtime.php'),
             ], 'bref-runtime');
         }
+
+        $dispatcher->listen(fn (JobProcessing $event) => $logManager->info("Processing job {$event->job->getJobId()}", ['name' => $event->job->resolveName()]));
+        $dispatcher->listen(fn (JobProcessed $event) => $logManager->info("Processed job {$event->job->getJobId()}", ['name' => $event->job->resolveName()]));
+        $dispatcher->listen(fn (JobExceptionOccurred $event) => $logManager->info("Job failed {$event->job->getJobId()}", ['name' => $event->job->resolveName()]));
```

Comment:
This should perhaps be in a separate package / service provider / user land

Reply:
Let's keep it here until someone complains.